src/screens/TransactionDetailsScreen.tsx

```diff
@@ -0,0 +1,180 @@
+// @ts-nocheck
+import moment from 'moment';
+import React, { Component } from 'react';
+import { View, StyleSheet, Text, Linking } from 'react-native';
+import { NavigationScreenProps, NavigationInjectedProps } from 'react-navigation';
+
+import { images } from 'app/assets';
+import { Image, Header, StyledText, Chip, ScreenTemplate } from 'app/components';
+import { CopyButton } from 'app/components/CopyButton';
+import i18n from 'app/locale';
+import { typography, palette } from 'app/styles';
+
+import BlueApp from '../../BlueApp';
+
+function onlyUnique(value, index, self) {
+  return self.indexOf(value) === index;
+}
+
+function arrDiff(a1, a2) {
+  const ret = [];
+  for (const v of a2) {
+    if (a1.indexOf(v) === -1) {
+      ret.push(v);
+    }
+  }
+  return ret;
+}
+
+type Props = Partial<NavigationScreenProps>;
+
+interface State {
+  isLoading: boolean;
+  tx: any;
+  from: any[];
+  to: any[];
+  wallet: any;
+}
+
+export class TransactionDetailsScreen extends Component<Props, State> {
+  static navigationOptions = (props: NavigationScreenProps) => {
+    const transaction = props.navigation.getParam('item');
+    return {
+      header: <Header navigation={props.navigation} isBackArrow title={moment.unix(transaction.time).format('lll')} />,
+    };
+  };
+
+  constructor(props: Props) {
+    super(props);
+    const { hash } = props.navigation.getParam('item');
```

Comment:
Can you introduce model for `Transaction`? would be useful :)

and:
- name the navigation param as `transaction`, not 'item'
add typings for
```
type Props = NavigationInjectedProps<{transaction: Transaction}>>; 
```
same typings for 
```
static navigationOptions = (props: NavigationScreenProps) => {
```